### DISCUSSION TITLE
Not able to add an organisation

### DISCUSSION THREAD

**Discussion by MarloesHooghiemstra** [alkem-io/alkemio#950] [Q&A]
Hi, I'm trying to add organizations to my profile. But I'm unable to add TU Delft and Digicampus as an organization. I only see the message: Geen gerelateerde organisaties beschikbaar. Is there a bug, or is it simply something I didn't find yet to add organizations?

**Comment by Comoque1** [comment]
Hello Marloes,

Thank you for sharing your experience!

For the moment, the ways to become member of an Organization are as follows:
1. Ask the Organization admin
2. If organization has the preference `Automatically add new users with emails matching the domain of the Organization` enabled. This approach didn't work for you, as the Organization didn't have the preference set to `true`.

I will check how to proceed further with adding you as a member to these organizations and will let you know.

**Comment by techsmyth** [comment]
Hi Marloes,

As Evgeni has responded, this is something that is an open issue at the moment. The item is captured on the backlog but not yet towards the top.

In the very near term (target is next week) we are aiming to allow users to send a message to an Organization, which is then received by  admins of that organization - so that would be the near term option. So we could even extend this to have an option to "request association with this organization" - @SimoneZaza good to chat this one through with you also...

To be followed up...

Cheers,

Neil
